src/mlui/enums/Adabound.py

```diff
@@ -0,0 +1,225 @@
+
+
+
+import tensorflow.compat.v2 as tf
+from keras import backend
+from keras.optimizers import Optimizer
+
+class AdaBound(Optimizer):
+    """
+    Adabound optimizer implementation 
+    # References
+        - [Adaptive Gradient Methods with Dynamic Bound of Learning Rate]
+          (https://openreview.net/forum?id=Bkg3g2R9FX)
+
+
+
+    Parameters
+    -----------
+        learning_rate : float 
+            Adam learning rate (default: 1e-3)
+
+        base_lr: float >= 0.
+             Used for loading the optimizer. Do not set the argument manually.    
+
+        
+        final_lr : float
+                  final (SGD) learning rate (default: 0.1)
+
+        beta1,beta2 : float between [0,1] 
+            coefficients used for computingrunning averages of gradient and its square (default: (0.9, 0.999))
+          
+        gamma : float
+                convergence speed of the bound functions (default: 1e-3)
+
+        eps : float
+            term added to the denominator to improve numerical stability (default: 1e-8)
+
+        weight_decay : float 
+             weight decay (L2 penalty) (default: 0)
```

Comment:
There should be a capital letter at the start, a dot at the end, a correct default value placement.